iOS/Flows/Conversation/ConversationCoordinator.swift

```diff
@@ -25,11 +25,11 @@ class ConversationCoordinator: PresentableCoordinator<Void> {
     }()
     private var cancellables = Set<AnyCancellable>()
 
-    var conversation: DisplayableConversation?
+    var conversation: Conversation?
```

Comment:
DisplayableConvesation and ConversationType are both gone. We now directly reference Stream ChatChannels, however I've typealiased it so that we use the "Conversation" semantics.